src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
+const ERROR_MESSAGE = 'Invalid values entered';
```

Comment:
👍 
But give more descriptive name for this error message(what type of error message it is)